### ISSUE TITLE
-Infinity times -Infinity is Infinity, not -Infinity.

### ISSUE THREAD

**Issue by James103** [Patashu/break_eternity.js#172]
Currently, `new Decimal(-Infinity).times(-Infinity)` returns the Decimal equivalent of `-Infinity`, instead of the correct result being a Decimal representation of `Infinity`. The analogue with JavaScript numbers, `-Infinity * -Infinity`, returns positive `Infinity` as expected.

```js
new Decimal(-Infinity).times(-Infinity).toString() // -Infinity
Math.sign(-Infinity * -Infinity) // 1
Decimal.sign(-Infinity * -Infinity) // 1
Decimal.sign(Decimal.times(-Infinity, -Infinity)) // -1
```

**Comment by MathCookie17** [comment]
I'll fix this one whenever I get around to updating break_eternity again.

**Comment by MathCookie17** [comment]
Fixed in v2.1.1.
